WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Dynamic/DynamicDashboardCard.swift

```diff
@@ -39,29 +40,26 @@ struct DynamicDashboardCard: View {
         }
         .padding(.bottom, Length.Padding.single)
         .padding(.horizontal, Length.Padding.double)
-        .fixedSize(horizontal: false, vertical: true)
     }
 
     @ViewBuilder
     var featureImage: some View {
         if let featureImageURL = input.featureImageURL {
             AsyncImage(url: featureImageURL) { phase in
-                Group {
-                    if let image = phase.image {
-                        image
-                            .resizable()
-                            .aspectRatio(contentMode: .fit)
-                    } else {
-                        Color.DS.Background.secondary
-                            .frame(maxWidth: .infinity)
-                            .frame(height: 150)
-                    }
+                switch phase {
+                case .success(let image):
+                    image
+                        .resizable()
+                        .aspectRatio(input.featureImageWidthToHeightRatio, contentMode: .fit)
+                        .clipShape(
+                            RoundedRectangle(
+                                cornerRadius: Length.Radius.small
+                            )
+                        )
```

Comment:
[Suggestion] @alpavanoglu I believe we need to apply the clip shape to both the image and its placeholder to maintain consistency. Without it, the placeholder will appear as a rectangle, and only when the image loads will the corners become rounded.